onnx/common/ir.h

```diff
@@ -985,6 +989,11 @@ friend struct Value;
   Value * addInput() {
     return input_->addOutput();
   }
+
+  void popInput() {
+    input_->popOutput();
```

Comment:
confusing function call...  popoutput from input :).

Reply:
I also felt confused for the original `addInput`...but the behavior is correct according to the original design.
I just added some comments for these two functions and hope it can resolve the confusion.
Thank you for bringing this up!